src/main/groovy/org/gradle/android/workarounds/room/RoomExtension.groovy

```diff
@@ -0,0 +1,24 @@
+package org.gradle.android.workarounds.room
+
+import org.gradle.android.workarounds.room.task.MergeAssociations
+import org.gradle.api.file.Directory
+import org.gradle.api.file.DirectoryProperty
+import org.gradle.api.model.ObjectFactory
+import org.gradle.api.provider.Provider
+
+import javax.inject.Inject
+
+abstract class RoomExtension {
```

Comment:
let's add the `@CompileStatic` flag on this class and all the other added classes here if possible!

Reply:
added `@CompileStatic` in all the new classes